ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -27,11 +27,12 @@
               role="menu">
             <li ng-repeat="tier in jobFilters.tiers">
               <div class="checkbox dropdown-link">
-                <label>
+                <label title="{{(isSingleTierSelected && tiers[tier] == true) ? 'Must have at least one tier selected at all times' : ''}}">
```

Comment:
so here you could just call the function as ``isSingleTierSelected()`` instead of ``isSingleTierSelected``.